resources/de-amp.js

```diff
@@ -0,0 +1,56 @@
+if (deAmpEnabled) {
```

Comment:
I'd prefer to keep these in an IIF if possible, for consistency with uBO. It's probably fine for now, but if there ends up being an upstream change to the injections it'd be nice to keep our own required changes to a minimum.

Reply:
👍